src/Components/benchmarkapps/BlazingPizza.Server/ssr-benchmarks.yml

```diff
@@ -0,0 +1,81 @@
+imports:
+    - https://raw.githubusercontent.com/dotnet/crank/main/src/Microsoft.Crank.Jobs.Bombardier/bombardier.yml
+
+jobs:
+  server:
+    source:
+      localFolder: ../../../../artifacts/bin/BlazingPizza.Server/Release/net8.0
+
+scenarios:
+  ssr:
+    application:
+      job: server
+      framework: net8.0
+      runtimeVersion: 8.0.0-preview.2.23125.5
+      executable: dotnet
+      arguments: BlazingPizza.Server.dll
+    load:
+      job: bombardier
+      variables:
+        serverPort: 5000
+        path: /
+
+profiles:
```

Comment:
You can add this file in the imports instead of adding the following section
https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/aspnet.profiles.standard.yml

It contains the definitions for all our machines.

Reply:
Good tip - thanks!